function foo() {
    try {
        throw 1;
    } catch (e) {

    }
}

foo();
